xrtpy/response/effective_area.py

```diff
@@ -106,7 +105,7 @@ def ccd_observation_date_to_seconds(self):
             t0 = _ccd_contamination_file_time[0]
             dt = time - t0
             ccd_observation_date_to_seconds.append(
-                (self.observation_date + timedelta(0, dt)).strftime("%s")
+                (self.observation_date + timedelta(0, dt)).strftime("%S")
```

Comment:
For some reason, the lowercase `%s` seemed to be working fine on Linux but not on Windows.  Not sure why!

Reply:
The (s)mall details.... Thanks for the update to the code Nick!